### PR TITLE
Task: finding the substring with even idices was added

### PR DESCRIPTION
**Pull request by ZheltokDina**

### CODE REVIEW

File: taskDEV2/EntryPoint.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  /// </summary>
+  class EntryPoint
+  {
+    static void Main(string[] args)
+    {
+      try
+      {
+        Console.WriteLine("Please, enter a string:");
+        string sequenceOfSymbols = Console.ReadLine();
+        StringCounter counter = new StringCounter(sequenceOfSymbols);
```
Comment (IgorKheidorov) [inline taskDEV2/EntryPoint.cs]: Indexes

File: taskDEV2/StringCounter.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Text;
+
+namespace taskDEV2
+{
+  /// <summary>
+  /// This class finds substring with even idices
+  /// </summary>
+  class StringCounter
```
Comment (IgorKheidorov) [inline taskDEV2/StringCounter.cs]: Bad name, why Counter?

File: taskDEV2/StringCounter.cs [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    public StringCounter(string sequenceOfSymbols)
+    {
+      this.SequenceOfSymbols = sequenceOfSymbols;
+    }
+
+    /// <summary>
+    /// This method finds substring with even idices
+    /// </summary>
+    /// <returns>substring with even idices </returns>
+    public string FindASubstringWithEvenIdices()
```
Comment (IgorKheidorov) [inline taskDEV2/StringCounter.cs]: Not find string, create string

File: taskDEV2/StringCounter.cs [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    {
+      StringBuilder line = new StringBuilder();
```
Comment (IgorKheidorov) [inline taskDEV2/StringCounter.cs]: why line? bad name

File: taskDEV2/StringCounter.cs [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+      for (int indexOfSymbol = 0; indexOfSymbol < SequenceOfSymbols.Length; indexOfSymbol++)
```
Comment (IgorKheidorov) [inline taskDEV2/StringCounter.cs]: Much more simple i+=2 and no need to calculate index each time, not effective

File: taskDEV2/Properties/AssemblyInfo.cs [hunk 6]
```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
```
Comment (IgorKheidorov) [inline taskDEV2/Properties/AssemblyInfo.cs]: russian version, reinstall

Comment (ZheltokDina) [reply taskDEV2/Properties/AssemblyInfo.cs]: I reinstalled to english version

File: taskDEV2/EntryPoint.cs [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+  /// </summary>
+  class EntryPoint
+  {
+    static void Main(string[] args)
+    {
+      try
+      {
+        Console.WriteLine("Please, enter a string:");
+        string sequenceOfSymbols = Console.ReadLine();
+        StringCreater creater = new StringCreater(sequenceOfSymbols);
```
Comment (IgorKheidorov) [inline taskDEV2/EntryPoint.cs]: Spelling

File: taskDEV2/StringCreater.cs [hunk 8]
```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Text;
+
+namespace taskDEV2
+{
+  /// <summary>
+  /// This class finds substring with even idices
+  /// </summary>
+  class StringCreater
```
Comment (IgorKheidorov) [inline taskDEV2/StringCreater.cs]: Name of class, creator

File: taskDEV2/StringCreater.cs [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+    public StringCreater(string sequenceOfSymbols)
+    {
+      this.SequenceOfSymbols = sequenceOfSymbols;
+    }
+
+    /// <summary>
+    /// This method finds substring with even idexes
+    /// </summary>
+    /// <returns>substring with even idexes </returns>
+    public string CreateASubstringWithEvenIdices()
```
Comment (IgorKheidorov) [inline taskDEV2/StringCreater.cs]: better GetSubstring...

File: taskDEV2/StringCreater.cs [hunk 10]
```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Text;
+
+namespace taskDEV2
+{
+  /// <summary>
+  /// This class finds substring with even idices
```
Comment (IgorKheidorov) [inline taskDEV2/StringCreater.cs]: Not find, create

File: taskDEV2/StringCreator.cs [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+    }
+
+    /// <summary>
+    /// This method get a substring with even indexes
+    /// </summary>
+    /// <returns>substring with even indexes </returns>
+    public string GetASubstringWithEvenIndexes()
+    {
+      StringBuilder substringWithEvenIndexes = new StringBuilder();
+      for (int indexOfSymbol = 0; indexOfSymbol < SequenceOfSymbols.Length; indexOfSymbol += 2)
```
Comment (AlexanderSoroka) [inline taskDEV2/StringCreator.cs]: just i is enough

File: taskDEV2/StringCreator.cs [hunk 12]
```diff
@@ -0,0 +1,51 @@
...
+      get
+      {
+        return sequenceOfSymbols;
+      }
+      set
+      {
+        sequenceOfSymbols = value;
+        if (string.IsNullOrEmpty(sequenceOfSymbols))
+        {
+          throw new Exception("Your string is empty.");
```
Comment (AlexanderSoroka) [inline taskDEV2/StringCreator.cs]: wrong exception type

**Pull request merged**